src/axom/sidre/docs/sphinx/data_vs_metadata.rst

```diff
@@ -239,44 +236,54 @@ The output of the code is::
        	Base address of array in A group: 0xc595b8
        	Base address of array in A group: 0xc595b8
 
-  After destroyGroup(A_grp) call:
-       	Num buffers in datastore: 1
-        Is buffer allocated? 1 
-        Num views attached to buffer: 2
+  After destroyGroup(A) call:
        	Num views in group B: 1
        	aview2 in B group has values:	1   3   5   7   9
+       	Num buffers in datastore: 1
+        Is buffer allocated? 1 
+        Num views attached to buffer: 1
 
-  After destroyGroup(B_grp) call:
+  After destroyGroup(B) call:
         Num buffers in datastore: 1
         Is buffer allocated? 1
-        Num views attached to buffer: 2
+        Num views attached to buffer: 0
   
+The last operation in this example is intended to emphasize the explicit 
+nature of Sidre methods. In particular, when a group is destroyed, its views 
+are also destroyed, but their *data may remain intact*. As shown earlier, 
```

Comment:
"their *data may remain intact*."  This might be a good point to discuss why the data may remain intact.